temba/channels/tests.py

```diff
@@ -9458,6 +9584,55 @@ def test_send_media(self):
             self.assertFalse(ChannelLog.objects.filter(description__icontains="local variable 'response' "
                                                                               "referenced before assignment"))
 
+    def test_send_quick_replies(self):
+        joe = self.create_contact("Joe", urn="facebook:1234")
+        metadata = dict(quick_replies=['Yes', 'No'])
+        msg = joe.send("Facebook Msg", self.admin, trigger_send=False, quick_replies=metadata)[0]
+
+        with self.settings(SEND_MESSAGES=True):
```

Comment:
Same thing here, let's be consistent in using `@override_settings` for this instead